docs/user-guide.md

```diff
@@ -31,11 +45,22 @@ The partner can select the language of their preference from the dropdown in the
 * French
 * Arabic
 
+_Note_: Once a partner has registered, they must go into the partner management portal to access partner details.
+
+
 ![CTK login page](\_images/ctk-toolkit-login.png)
 
+## Terms and Conditions Consent
+
+The Terms & Conditions popup window will appear for the partner after they log in. It has to be accepted by the partner. If the partner does not accept the Terms & Conditions, they will be logged out.
+
+Also, upon logging in, partners will be prompted with a popup if there have been any changes to the Terms & Conditions since their last consent.
+
+ ![CTK consent window](\_images/ctk-consent-popup.png)
+
 ## Dashboards
 
-Once the partner logs into the compliance tool kit, they can view the `Project Dashboard` by default and would have the option to view the `Biometric Data Dashboard` and `My Reports Dashboard` as well.
+Once the partner accepts the Terms & Conditions, they can view the `Project Dashboard` by default and would have the option to view the `Biometric Data Dashboard` and `My Reports Dashboard` as well.
```

Comment:
After accepting the Terms & Conditions, partners gain access to the Project Dashboard by default. Additionally, they have the option to explore the Biometric Data Dashboard and My Reports Dashboard.